patches/minecraft/net/minecraft/init/Bootstrap.java.patch

```diff
@@ -10,10 +10,11 @@
          BlockDispenser.field_149943_a.func_82595_a(Items.field_151133_ar, new BehaviorDefaultDispenseItem()
          {
              private final BehaviorDefaultDispenseItem field_150840_b = new BehaviorDefaultDispenseItem();
-@@ -496,6 +499,7 @@
+@@ -496,6 +499,8 @@
              StatList.func_151178_a();
              Biome.func_185358_q();
              func_151353_a();
++            net.minecraftforge.fluids.FluidRegistry.registerFluids();
```

Comment:
I suggest to make some method like `ForgeRegistries.initForgeRegistries` to prevent future increase in patch size.